### PR TITLE
Update and rename comparison.md to flame.md

### PR DESCRIPTION
**Pull request by Xenthys**
At least be honest lol, updated the description to reflect your original commit message since you don't even try to understand why PHP is adapted for most projects you've listed. You could've asked if you were really curious, cheers!

### CODE REVIEW

**Comment by diced** [comment]
Good idea, I was going to add more of a better comparison, yet I just wanted to get something out pretty quick, and i'll probably be more honest in that. And now since I am pretty curious, please explain!
> you don't even try to understand why PHP is adapted for most projects you've listed. 

(I'll be closing this later)

**Comment by Xenthys** [comment]
Oh nice, I'll be honest and say I am pleasantly surprised by your reply!

I can't speak for all the uploaders of course, but the reason I picked PHP despite knowing other languages is due to the versatility it offers: most people looking for a quick uploader are not willing to shell out money for a full VPS, and even less learn how to configure it properly and securely. We had a lot of people coming on the ShareX Discord server with random PHP scripts they found here and there that offered absolutely no security, so if they were gonna go that way at least we could offer something easy to setup that only requires half a brain and not even a database to install while offering a reasonable security for multiple users.

Many of these users are alone or have a few friends. They only rent a very cheap web hosting service and sometimes it's even some free offer on a random host that will probably disappear 6 months later. It doesn't allow them to run anything that isn't a PHP script. They could use their FTP access, some do, but it's kinda hard to share if you're not alone and want to maintain some kind of privacy, plus if you don't like PHP I believe you already know why FTP can be an even worse idea.

The best way is probably for each user to create their own free S3 bucket with any other provider having a free tier, but you lose the vanity element that is a shared custom domain name for everyone unless you all upload to the same bucket… which has the same issue as the aforementioned FTP method. It also requires each user to create an account and give their payment details to said provider, then configuring a bucket, etc. As you made an uploader yourself, I believe you see why it can be more convenient for your friends to just give them an API key and a readily available `.scxu` so they can use your service straight away.

**TL;DR: it's a usability issue. My uploader was made with cheap/free web hosting services in mind that may not even offer a database of any kind. Grab the script, edit the first lines, drag & drop, ready to go.** As much as I recognize running on the V8 engine is so much better for real projects in case you want to run a public service that can potentially be scaled or just host hundreds of users, it's overkill and unusable for random users with random web hosting services who want to share their access with two friends who will upload something like 50 screenshots per month.

I'm available on Discord if you want to pursue this discussion, you'll easily find me in the aforementioned ShareX server if needed of course. All in all, different goals and features in mind, I'm glad there is also a project like yours for people who need it! You may close this issue at any time, please ensure you really publish a fair comparison though. ;)

**Comment by diced** [comment]
Thanks for replying, as much as your claim of usability is very true, the reason I created Zipline was to offer a highly configurable image uploader with a *somewhat* nice looking front-end, as many of these previous uploaders were somewhat lacking. Zipline can be easily run by just getting the example config, then editing the values to your liking, and then just running the docker-compose (or not since it's not required to use docker), like what is said on the getting-started documentation. In my opinion I'd (and many of my friends) rather have a nice configuration file than editing lines in some code. I do think having a no database option is great for Zipline so I might consider it for it's next release.

When you say the V8 is overkill/unusable, it's like saying any sort of API that uses Node is somewhat overkill and unusable, which isn't really the case... Zipline is completely usable too, on my server its been running for almost a month, and doesn't use that much resources.

With all that said, I will make sure to get a better comparison between these uploaders soon.
